### PR TITLE
plugin: Sigma

### PR DESCRIPTION
**Pull request by monesga**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/monesga/obsidian-sigma

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [x]  Linux
  - [x]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by brimwats** [comment]
you forgot the link to the plugin!

**Comment by monesga** [comment]
> 

Sorry about that. I updated the submission comment with a link to the plugin

**Comment by joethei** [comment]
You can remove `obsidian` from both the plugin id an name, its unecesarry.

["version": "1.0.2s",](https://github.com/monesga/obsidian-sigma/blob/8083ee46caf1a6983d6ca9a6507a0a065c20348b/manifest.json#L4)
Version numbers cannot contain additional characters, you need to remove the `s` for the plugin to be downloadable.(this seems to be a typo).

[// This adds a status bar item to the bottom of the app. Does not work on mobile apps.](https://github.com/monesga/obsidian-sigma/blob/8083ee46caf1a6983d6ca9a6507a0a065c20348b/main.ts#L584-L586)
Any particular reason yu are doing this?
I don't see any reason for it.

**Comment by monesga** [comment]
These issues are addressed in latest release

File: community-plugins.json [hunk 1]
```diff
@@ -5402,5 +5402,12 @@
         "author": "Abu Ibrahim",
         "description": "Easily add Quran verses and translations in Obsidian with this text replacement plugin",
         "repo": "abuibrahim2/quranlookup"
+    },
+    {
+        "id": "obsidian-sigma",
```
Comment (liamcain) [inline community-plugins.json:5407]:
```suggestion
        "id": "sigma",
```

**Pull request merged**